src/expo-plugins/index.ts

```diff
@@ -0,0 +1,140 @@
+import {
+  ConfigPlugin,
+  createRunOncePlugin,
+  withAppDelegate,
+  AndroidConfig,
+  withMainApplication,
+  withAndroidManifest,
+  withInfoPlist,
+} from '@expo/config-plugins';
+import {
+  addImports,
+  appendContentsInsideDeclarationBlock,
+} from '@expo/config-plugins/build/android/codeMod';
+import {
+  addObjcImports,
+  insertContentsInsideObjcFunctionBlock,
+} from '@expo/config-plugins/build/ios/codeMod';
+import type { IntercomPluginProps, IntercomRegion } from './@types';
+import packageJson from '../../package.json';
+
+const mainApplication: ConfigPlugin<IntercomPluginProps> = (_config, props) =>
+  withMainApplication(_config, (config) => {
+    let stringContents = config.modResults.contents;
+    stringContents = addImports(
+      stringContents,
+      ['com.intercom.reactnative.IntercomModule;'],
+      false
+    );
+    stringContents = appendContentsInsideDeclarationBlock(
+      stringContents,
+      'onCreate',
+      `IntercomModule.initialize(this, "${props.androidApiKey}", "${props.appId}");`
+    );
+    config.modResults.contents = stringContents;
+    return config;
+  });
+
+const androidManifest: ConfigPlugin<IntercomPluginProps> = (
+  _config,
+  { intercomRegion = 'US' }
+) => {
+  let newConfig = AndroidConfig.Permissions.withPermissions(
+    _config,
+    [
+      'android.permission.READ_EXTERNAL_STORAGE',
+      'android.permission.VIBRATE',
+    ].filter(Boolean)
+  );
+
+  newConfig = withAndroidManifest(newConfig, (config) => {
+    const currentMainApplication =
+      AndroidConfig.Manifest.getMainApplicationOrThrow(config.modResults);
+    const androidRegionMapper: Record<IntercomRegion, string> = {
+      US: '@integer/intercom_server_region_us',
+      EU: '@integer/intercom_server_region_eu',
+      AU: '@integer/intercom_server_region_au',
```

Comment:
```suggestion
      AU: '@integer/intercom_server_region_aus',
```

For android (australia) the config needs to be:
```xml
<meta-data
   android:name="io.intercom.android.sdk.server.region"
   android:value="@integer/intercom_server_region_aus" 
/>
```